boms/modeshape-bom-embedded/pom.xml

```diff
@@ -366,7 +366,7 @@
             </dependency>
             <!--Required by infinispan-cachestore-jdbc -->
             <dependency>
-                <groupId>c3p0</groupId>
+                <groupId>com.mchange</groupId>
```

Comment:
no, in CR1 ISPN was using 0.9.5-pre-X which was a bit shabby. So I postponed changing our version until ISPN uses a "final" c3p0 version.